src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
The id of the default tag is also used to determine the visibility of the reset button in that file, so it is still needed there. Importing from a common module sorts that out though.